### ISSUE TITLE
Adaptec 71605

### ISSUE THREAD

**Issue by RacerXXX** [fbelavenuto/arpl#733]
Hi 
I did some testing today with DS3622xs.  The install worked but does not detect (it's not in the modules with the current build)  Can you add it in for test?

For reference
https://raw.githubusercontent.com/pocopico/rp-ext/master/aic94xx/rpext-index.json

**Comment by maomaoaigou** [comment]
Yes, my Adaptec 71605 is not recognised either. Have you found a solution?
